libethereum/Executive.cpp

```diff
@@ -165,16 +165,15 @@ void StandardTrace::operator()(uint64_t _steps, uint64_t PC, Instruction inst, b
 
 string StandardTrace::json(bool _styled) const
 {
-	if (m_trace.empty())
-		return string();
+    if (m_trace.empty())
+        return {};
```

Comment:
Here wasn't anything wrong, but `string{}` syntax is preferred to `string()` because of some C++ parser edge cases.
I also omitted the type `string` because the compiler will auto-deduct it.

Reply:
Thanks, will keep this and the clang-format in mind for the future.